src/Analysis/GWASUIMockup/utils.js

```diff
@@ -0,0 +1,754 @@
+export const mockCohortData = [
```

Comment:
remove these mock data if they arfe not being used any more

Reply:
removed